vuepress/docs/next/tutorials/devops/change-default-datasource.md

```diff
@@ -77,77 +56,77 @@ container.
-of the sample project. 
+## Modify your EntandoApp 
 
-* Firstly, you would need to change the variable `server.base.image` to reflect the fully
-qualified URI of your Docker image for EAP and/or Wildfly. For example, if you decided to extend the Wildfly image,
-go to the Maven profile with the id `wildfly` and change the  `server.base.image` to look like this:
+Modify your EntandoApp project as described in the [Building a Docker Image for the Entando Core tutorial](./build-core-image.md). It will require two changes to the pom.xml file of the sample project. 
 
-`<server.base.image>your-docker-registry.com/your-org/your-base-image:1.0.0</server.base.image>`
+1. First, change the variable `server.base.image` to reflect the fully qualified URI of your Docker image for Tomcat. For example, if you decided to extend the Tomcat image, go to the Maven profile with the id `tomcat` and change the `server.base.image` to this:
+``` bash
+<server.base.image>YOUR-DOCKER-REGISTRY.com/YOUR-ORG/YOUR-BASE-IMAGE:1.0.0</server.base.image>
+```
 
-* Then you need to change the `<from>` element in the Fabric8 Maven Docker Plugin to reflect this variable.
+2. Then, change the `<from>` element in the Fabric8 Maven Docker Plugin to reflect this variable.
 Navigate to the first `<image>` element in the pom.xml file. You have found the correct one
 if it contains the following xml:
+```
+<from>entando/${server.base.image}:${entando.version}</from>
+```
 
-`<from>entando/${server.base.image}:${entando.version}</from>`
-
-Delete the `entando/` prefix and the `:${entando.version}` suffix:
+3. Delete the `entando/` prefix and the `:${entando.version}` suffix:
 
 `<from>${server.base.image}</from>`
 
-Please note that, should you still require support for both Wildfly and EAP in your Maven pom, you may need to 
-repeat this exercise for both the `eap` and `wildfly` profiles. 
-
-### 5. Build and push your custom Docker image
-
-Before building your Docker image, it would be a good idea to change the name of the image you want to build. Navigate
-to the previously modified `<image>` element in the Fabric8 Maven Docker Plugin.You can change the `<name>` element
-to reflect your preferred name, e.g.
-
-`<name>your-docker-registry.com/your-org/your-entando-app:1.0.0</name>`
+### Build and push your custom Docker image
 
-Run:
-
-`mvn clean package -Pwildfly -Pderby`
-
-A new Docker image should now be available named `your-docker-registry.com/your-org/your-entando-app:1.0.0`
+1. Before building your Docker image, it is a good idea to change the name of the image you want to build. Navigate
+to the previously modified `<image>` element in the Fabric8 Maven Docker Plugin. You can change the `<name>` element
+to reflect your preferred name:
+e.g.,  
+``` bash
+<name>YOUR-DOCKER-REGISTRY.com/YOUR-ORG/YOUR-ENTANDO-APP:1.0.0</name>
+```
+**Does this work for the tomcat version?**
+2. Run:
+``` bash
+mvn clean package -Ptomcat -Pderby
```

Comment:
@pietrangelo does the run command above work for the  tomcat version?

Reply:
I don't see a tomcat profile - https://github.com/entando/app-engine/blob/develop/webapp/pom.xml. @eugeniosant thoughts?